images/toolbox/Dockerfile-windows

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Should be 1809 or ltsc2022
+ARG WINDOWS_VERSION
+
+# See https://kubernetes.io/docs/concepts/windows/intro/#pause-container
+# Unlike the image from registry.k8s.io, there are also different named tags for different OS
+# versions, so we have more control.
+FROM mcr.microsoft.com/oss/kubernetes/pause:3.9-windows-${WINDOWS_VERSION}-amd64
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A Docker image with useful tools (for networking) based on the K8s pause image"
+
+# We do not add any tools to the image at the moment.
+# The image comes with pause + wincat + anything included by default in the nanoserver base image.
```

Comment:
Does it have ping?

Reply:
Yes, I checked. It's included by default in the nanoserver image